how-tos/testing-your-backend.mdx

```diff
@@ -0,0 +1,84 @@
+---
+title: "Testing Your Backend"
+
+---
+
+When your frontend makes a request to your backend, it [includes an access token](https://docs.propelauth.com/how-tos/making-authenticated-requests). But, what if we don’t have a frontend yet? Or what if we want to test our backend independently from our frontend?
+
+The following guide will walk you through how to use and test the PropelAuth backend libraries in isolation from your frontend.
+
+## Getting an Access Token Without a Frontend
+
+There are a few different ways that you can get started with initializing this process, but they all boil down to creating an [Access Token](https://docs.propelauth.com/overview/access-tokens) for use in your testing.
+
+With our frontend libraries, you can get an access token and send them to the backend via an Authorization header. We do also have an endpoint on our backend libraries that can create an access token for one of your users with a configurable expiration time. 
+
+Now to create this token, you could use the command line, which might look something like:
+
+```bash
+curl -H "Content-Type: application/json" 
+     -H "Authorization: Bearer {PROPELAUTH_API_KEY}" 
+     -X POST 
+     -d '{"user_id": "{USER_ID}", duration_in_minutes: 1440}'
+     "{AUTH_URL}/api/backend/v1/access_token" 
+```
+
+You could also use something like the Python interpreter and one of our Python libraries:
+
+```python
```

Comment:
This probably won't render well no matter what we choose since it's a mix of python and regular text, but maybe just text is better?